Firestore/core/src/local/leveldb_lru_reference_delegate.cc

```diff
@@ -147,7 +147,7 @@ int LevelDbLruReferenceDelegate::RemoveOrphanedDocuments(
 
 int LevelDbLruReferenceDelegate::RemoveTargets(
     ListenSequenceNumber sequence_number, const LiveQueryMap& live_queries) {
-  return db_->target_cache()->RemoveTargets(sequence_number, live_queries);
+  return (int)db_->target_cache()->RemoveTargets(sequence_number, live_queries);
```

Comment:
We disallow C-style casts in C++. Instead of `(int)x`, use `static_cast<int>(x)`. See the [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Casting).